doc/changes/latest.inc

```diff
@@ -24,6 +24,8 @@ Current (1.1.dev0)
 Enhancements
 ~~~~~~~~~~~~
 
+- Add built-in sensor layout files for Geodesic Head Web 130 and 280 devices (:gh:`10627` by `Evan Hathaway`_)
```

Comment:
@ephathaway can you insert below the items from new contributors? We try to keep them on top of the list.